docs/docs/documentation/getting-started/installation/postgres.md

```diff
@@ -55,13 +55,17 @@ services:
     container_name: postgres
     image: postgres
     restart: always
+    volumes:
+      - mealie-pgdata:/app/data/
```

Comment:
Should this be?

```yaml
- mealie-pgdata:/var/lib/postgresql
```

Reply:
Yeah, of course, now i didnt pay attention myself...
Will fix that in a moment